braindecode/models/eegsimpleconv.py

```diff
@@ -0,0 +1,148 @@
+"""
+EEG Simple Convolutional Neural Network from Yassine El Ouahidi et al. (2023).
+
+Each EEG channels is convoled individually.
+
+"""
+# Authors: Yassine El Ouahidi <eloua.yas@gmail.com>
+#
+# License: BSD-3
+
+import torch
+
+from . import Resample
+from .base import EEGModuleMixin
+
+
+class EEGSimpleConv(EEGModuleMixin, torch.nn.Module):
+    """EEGSimpleConv from [Yassine2023]_.
+
+    Can you increase the description here Yassine?
```

Comment:
Can you add more information in the description @elouayas?

Reply:
Good examples in EEG Conformer and Labram

Reply:
Done ! 
Thanks for the suggestions !